content/nginx-one/how-to/certificates/manage-certificates.md

```diff
@@ -14,13 +14,31 @@ weight: 100
 
 This guide explains how you can manage SSL/TLS certificates with the F5 NGINX One Console. Valid certificates support encrypted connections between NGINX and your users. 
 
+You may have separate sets of SSL/TLS certificates, as described in the following table: 
+
+{{<bootstrap-table "table table-striped table-bordered">}}
```

Comment:
I don't think you need to use the bootstrap shortcode at all anymore: better defaults were added to the theme for tables.

Reply:
```suggestion
```